packages/server/test/question.integration.ts

```diff
@@ -382,8 +448,17 @@ describe('Question Integration', () => {
         userId: user.id,
         courseId: queue.courseId,
       });
+      const questionTypes = [];
+      forEach(QuestionTypes, async (questionType) => {
```

Comment:
I don't think foreach works weel with await function, I could be mistaking, but there is an occasional bug where not all items can be added/updated to database. i think we should use for loop?

Reply:
forEach allows for async, regular for loop does not